### DISCUSSION TITLE
404, message='Not Found', url='http://192.168.0.105:11434/api/chat'

### DISCUSSION THREAD

**Discussion by shenlaoma** [open-webui/open-webui#6065] [General]
Network Environment:

The entire setup uses a local network, with all hosts and servers on the same subnet:
- Ollama (v0.3.12 on windows11) is running on 192.168.0.105, using the default port 11434.
- The open webui is deployed using Docker with the container IP: 192.168.0.3, and the port mapping is 3000->8080.

Additional Information: 
-Ollama is listening on 0.0.0.0, the model is running normally, and it can be called through other web UIs, [e.g., MaxKB], with the base URL "http://192.168.0.105:11434/" allowing for proper interaction.
-launch the command：
“ docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://192.168.0.105:11434/ -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main ”

Current Issue:
The open webui connects to the Ollama server without issues and can recognize the existing Ollama models. However, it cannot facilitate conversations, error msg：
”Ollama: 404, message='Not Found', url='http://192.168.0.105:11434/api/chat'“

![无标题](https://github.com/user-attachments/assets/78a20781-d229-479d-a5f5-bbd7bd84e53a)
![image](https://github.com/user-attachments/assets/2c0b7485-2f89-41c7-bf3b-41ac2bca0095)
![image](https://github.com/user-attachments/assets/4736797d-35f8-4eca-aa24-2e898603778d)

**Comment by shenlaoma** [comment]
This was an occasional bug. After I removed the image, pulled it again, and redeployed the webui, it was finally resolved.

**Reply by SwarmKit** [reply]
Hello! May I ask which version of the mirror you are using? I have also encountered this problem.
ollama version : 0.3.12

**Comment by SwarmKit** [comment]
I also have the same problem. I use the mirror ghcr.io/open-webui/open-webui:main. How can I solve it?

**Comment by hi-unc1e** [comment]
By using wireshark to capture the HTTP requests, I got  the root cause

404 errored command:
```
docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://10.211.55.2:11434/ -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```
![image](https://github.com/user-attachments/assets/5bb5c286-0981-4c90-82ca-24e1308f674b)


My solution is as follows, which has removed the ending slash -"/"
```
docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://10.211.55.2:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```
